docs/benchmarks/benchmarks.rst

```diff
@@ -10,12 +10,13 @@ Benchmarks
    tsunami
    mcmc_benchmark
    mice_benchmark
+   pivot_benchmark
    gkdr_benchmark
    histmatch_benchmark
 
 The code includes a series of benchmarks that illustrate various pieces of the implementation. Benchmarks
 can be run from the ``mogp_emulator/tests`` directory by entering ``make all`` or ``make benchmarks`` to
```

Comment:
I think this should nowadays be the ``mogp_emulator/benchmarks`` directory.

Reply:
Ah, yes, good catch. I have a separate change ready to go that re-does this file, so I'll fix this page up when I push those changes.